.github/workflows/rosa-cluster-auto-provision-on-schedule.yml

```diff
@@ -0,0 +1,36 @@
+name: ROSA Scheduled Create
+
+on:
+  schedule:
+    - cron: '0 7 * * *' # Runs every day at 7 AM UTC
+
+env:
+  OPENSHIFT_VERSION: 4.12.17
+
+jobs:
+
+  checkout:
+    name: Create ROSA cluster
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup ROSA CLI
+        uses: ./.github/actions/rosa-cli-setup
+        with:
+          aws-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-default-region: ${{ vars.AWS_DEFAULT_REGION }}
+          rosa-token: ${{ secrets.ROSA_TOKEN }}
+
+      - name: Create ROSA Cluster
+        run: ./rosa_create_cluster.sh
+        working-directory: provision/aws
+        env:
+          VERSION: ${{ env.OPENSHIFT_VERSION }}
+          CLUSTER_NAME: shared-crossdc
```

Comment:
Do we need this parameter? Without it, this would default to `gh-keycloak` which means we don't need to set `clusterName` param for each benchmark run.

Reply:
good catch, I will revert this to fallback to the default `CLUSTER_NAME`